docs/source/api/plotting/geodesics/geodesics_scatter.rst

```diff
@@ -28,5 +28,5 @@ Color
 
 This module contains the methods for static geodesic plotting using scatter plots.
 
-.. automodule:: einsteinpy.plotting.geodesics.geodesics_scatter
+.. automodule:: einsteinpy.plotting.senile.geodesics_scatter
```

Comment:
The path to the docs will also be changed!

Reply:
I think you didn't change! The file should lie in 

docs/source/api/plotting/senile/geodesics_scatter.rst